### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [yaoweipeng/yaoweipeng.github.io#1]
The user @carlsonsantana validate your site template "https://yaoweipeng.github.io" and found these accessibility errors:
* Page https://yaoweipeng.github.io/:
  * Issue *e624c6e6-d246-4755-9a4f-67c8eeec877f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.63:1. Recommendation: change text colour to #317ab9.
    * **Selector**: `html > body > div > div:nth-child(3) > div > div:nth-child(2) > p > small > a`
    * **Context**: `<a href="/about" title="About this site">About this site</a>`
  * Issue *50c87bbb-bf86-42df-a821-229eb8203a53*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.48:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div:nth-child(3) > div > div:nth-child(1) > p > small`
    * **Context**: `<small>© jmcglone 2016</small>`
  * Issue *dc129a57-6436-4b23-824c-89a1b1fcd322*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #949494.
    * **Selector**: `html > body > div > div:nth-child(1) > h3 > a`
    * **Context**: `<a href="/">Jonathan McGlone</a>`
  * Issue *5c2c2ba1-a859-4d1d-ac33-b949262c91e3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.63:1. Recommendation: change text colour to #317ab9.
    * **Selector**: `html > body > div > div:nth-child(1) > ul > li:nth-child(3) > a`
    * **Context**: `<a href="/contact" title="Contact">Contact</a>`
  * Issue *9b5d9741-5220-48ab-a944-777a0a4feb9f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.63:1. Recommendation: change text colour to #317ab9.
    * **Selector**: `html > body > div > div:nth-child(1) > ul > li:nth-child(2) > a`
    * **Context**: `<a href="/notes" title="Notes">Notes</a>`
  * Issue *2be69606-5996-4025-a0ca-3d856bd3e9ac*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.63:1. Recommendation: change text colour to #317ab9.
    * **Selector**: `html > body > div > div:nth-child(1) > ul > li:nth-child(1) > a`
    * **Context**: `<a href="/cv" title="CV">CV</a>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/16db5377-a8bb-421c-be42-a76f377bf837/).
